components/textbox/css/_mixin.scss

```diff
@@ -138,6 +138,18 @@
 }
 
 @mixin utrecht-textbox--html-input {
+  &[type="password" i] {
+    @include utrecht-textbox--password;
+  }
+  &[type="url" i],
+  [type="email" i] {
+    @include utrecht-textbox--url;
+  }
+  &[type="number" i],
+  [type="tel" i] {
```

Comment:
I think for `tel` and `email` the `&` prefix is missing in the selector.

And I think we can include `&[inputMode="numeric" i]`, `&[inputMode="decimal" i]` and `&[inputMode="tel" i]` too.